django/bosscore/models.py

```diff
@@ -288,6 +294,31 @@ class DownsampleStatus:
     downsample_status = models.CharField(choices=DOWNSAMPLE_STATUS_CHOICES, default=DownsampleStatus.NOT_DOWNSAMPLED, max_length=100)
     downsample_arn = models.CharField(max_length=4096, blank=True, null=True)
 
+    # Is this a public channel?
+    public = models.BooleanField(null=False, default=False)
+
+    class StorageType:
+        """
+        String values are actual values stored in DB.
+        """
+        SPDB = 'spdb'               # Original Boss storage format.
+        CLOUD_VOLUME = 'cloudvol'   # Princeton's Cloudvolume format.
+
+    STORAGE_TYPE_CHOICES = (
+        (StorageType.SPDB, 'Boss Spatial Database'),
+        (StorageType.CLOUD_VOLUME, 'CloudVolume'),
+    )
+
+    storage_type = models.CharField(choices=STORAGE_TYPE_CHOICES, default=StorageType.SPDB, max_length=30, null=False)
```

Comment:
Is storage type part of the channel object? So if I get a boss resource from when I making cvdb call can I do 
```python
channel = resource.get_channel()
vol = CloudVolume(channel.bucket, channel.cv_path)
```

Reply:
Yes you can.